sdk/schemaregistry/schema-registry/package.json

```diff
@@ -59,6 +57,16 @@
       }
     ]
   },
+  "//sampleConfiguration": {
+    "productName": "Azure Schema Registry",
+    "productSlugs": [
+      "azure",
+      "azure-schema-registry"
```

Comment:
Nor this one.

Reply:
Ok, I've set disableDocsMs for now. Do I need to also remove the productSlugs or is that good enough?